src/components/drawing/index.js

```diff
@@ -1270,7 +1270,7 @@ function getFullUrl(localId, gd) {
 
     var context = gd._context;
     var baseUrl = context._exportedPlot ? '' : (context._baseUrl || '');
-    return 'url(\'' + baseUrl + '#' + localId + '\')';
+    return 'url(' + baseUrl + '#' + localId + ')';
```

Comment:
Per https://github.com/plotly/plotly.js/issues/5685#issuecomment-848883869 I think it's safer to keep the quotes when `baseUrl` is truthy (noting that the `_exportedPlot` condition appropriately clears it vs `context._baseUrl`).